src/Questions/MultipleCheckboxes/index.js

```diff
@@ -81,7 +81,7 @@ const QuestionMultipleCheckboxes = ({ question, useForm }) => {
                       )}
                       {...register(question.name, {
                         ...question.registerConfig,
-                        onChange: option.disableOthers && disableOthers,
+                        onChange: option.disableOthers && disableOthers(option),
```

Comment:
Let's swap this to ternary fallback to avoid passing bool as an event handler.
```suggestion
                        onChange: option.disableOthers ? disableOthers(option) : undefined,
```

Reply:
solved!